src/main/java/trackr/model/menu/ItemCost.java

```diff
@@ -0,0 +1,57 @@
+package trackr.model.menu;
+
+import static java.util.Objects.requireNonNull;
+import static trackr.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Item's cost in the Item list.
+ * Guarantees: immutable; is valid as declared in {@link #isValidCost(String)}
+ */
+public class ItemCost {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Cost should only contain numbers, and it should be at most 4 digits long";
+    public static final String VALIDATION_REGEX = "^([1-9]{0,4})$";
+    public final float value;
```

Comment:
Use `Double` instead of float.

Reason:
Java will return the following
`float a = 5.00;`
```
Error:
|  incompatible types: possible lossy conversion from double to float
|  float a = 5.00;
|            ^—^
```

Reply:
Thank you I will implement after this PR merges